ISL.Reidentification.Core/Services/Foundations/AccessAudit/AccessAuditService.Validations.cs

```diff
@@ -52,6 +53,11 @@ private async ValueTask ValidateAccessAuditOnAddAsync(AccessAudit accessAudit)
                 (Rule: await IsNotRecentAsync(accessAudit.CreatedDate), Parameter: nameof(AccessAudit.CreatedDate)));
         }
 
+        private async ValueTask ValidateAccessAuditOnRetrieveById(Guid accessAuditId)
+        {
+            Validate(
+                (Rule: await IsInvalidAsync(accessAuditId), Parameter: nameof(AccessAudit.Id)));
+        }
```

Comment:
Can simplify like this as it is a one liner

```suggestion
        private async ValueTask ValidateAccessAuditOnRetrieveById(Guid accessAuditId) =>
            Validate((Rule: await IsInvalidAsync(accessAuditId), Parameter: nameof(AccessAudit.Id)));
```